extensions/azurePublishNew/src/node/processStatusTracker.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { ProcessStatus } from '@botframework-composer/types';
+import { v4 as uuid } from 'uuid';
+
+type ProcessStatusStart = Omit<Partial<ProcessStatus>, 'id' | 'projectId' | 'processName' | 'log'> & {
+  projectId: string;
+  processName: string;
+};
+
+type ProcessStatusUpdate = Pick<Partial<ProcessStatus>, 'status' | 'message' | 'config'>;
+
+/**
+ * Creates a tracker of ProcessStatus.
+ * The tracker avoids throwing errors to prevent status tracking from halting the process itself.
+ */
+export const createProcessStatusTracker = () => {
```

Comment:
it seems like most of the time there is only one process in this tracker. Should it be exporting one single tracker?
```export const tracker = createProcessStatusTracker();```

Reply:
I don't want to restrict it to a single tracker within the module itself.